app/models/finance/statement_line_item.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class Finance::StatementLineItem < ApplicationRecord
```

Comment:
I think would add a validation that there can not be another line items may not existing for that declaration except `pending_withdrawal` and `withdrawn`
It's a bit defensive but I'd rather proceed with caution

Reply:
sorry, i don't follow your sentence, could you re-phrase? or give an example

Reply:
2. when creating an `awaiting_clawback` it must have a similar "paid" line item 
3. when creating a `clawedback` line items it should check it has the corresponding `awaiting_clawback`

Reply:
the concepts of clawbacks have not been added yet. they do not exist yet in the system. these concepts will be introduced in the next iteration. therefore I cannot at the moment add any validation pertaining to clawbacks.

Reply:
i can however add validation to prevent a declaration from being attached twice to a statement(s)